test/langtools/tools/javac/Diagnostics/7039014/T7039014a.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * @test /nodynamiccopyright/
+ * @bug     7039014
+ * @summary Confusing error message for method conflict
+ * @author archiecobbs
```

Comment:
nit: we don't add the `@author` tag anymore, it has been kept only if the original test had it, usually very old tests.

Reply:
Thanks - removed.